opensrp-reveal/src/main/assets/json.form/family_member_register.json

```diff
@@ -281,6 +282,22 @@
           "err": "Number must be 9 digits and must start with 0."
         }
       },
+      {
+        "key": "sleeps_outdoors",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "spinner",
```

Comment:
@githengi , the family forms use spinners instead of radios. Should we standardize on one type across the app for "select_one" data entry types? If so, do you have a preference?

Reply:
we can use radios